community/cypher/cypher/src/test/scala/org/neo4j/cypher/ProfilerAcceptanceTest.scala

```diff
@@ -269,8 +269,7 @@ class ProfilerAcceptanceTest extends ExecutionEngineFunSuite with CreateTempFile
     val result = legacyProfile(query)
 
     // then
-    val expectedTxCount = 1 + // First tx used to compile the query
-                          10  // One per 10 rows of CSV file
```

Comment:
The comment here was wrong. The extra one was *not* a transaction used by the planner. The planner had already executed on line 264. The extra transaction was because of the offset issue with `PERIODIC COMMIT`.